### PR TITLE
Expose rendering flags

### LINKED ISSUE #17: Expose flags for page rendering
Have you considered exposing the different rendering flags to the `fpdf_view.FPDF_RenderPageBitmapWithMatrix(bitmap, _page, matrix, clipping, _flags_here_)` call in your `PageReader.GetImage()` method?  

I am most interested in being able to render the pdf with annotations enabled: `FPDF_ANNOT 0x01`.

### PR DESCRIPTION
**Pull request by bseay**
This exposes the rendering flags I mentioned in #17.  I've overloaded your `GetImage` methods to include the flags parameter.  
The flags can be used like this: 
```
var rawBytes = pageReader.GetImage(RenderFlags.RenderAnnotations | RenderFlags.Grayscale);
```

I added a unit test that verifies that the rendered image contains the yellow cloud annotation, and also added an example if you wanted to include it.

Fixes #17

### CODE REVIEW

File: src/Docnet.Tests.Integration/PageReaderTests.cs [hunk 1]
```diff
@@ -231,6 +232,22 @@ public void GetPageWidthOrHeight_WhenCalledWithScalingFactor_ShouldMach(Input ty
             });
         }
 
+        [Fact]
+        public void GetImage_WhenCalledWithRenderAnnotationsFlag_ShouldRenderAnnotation()
```
Comment (Modest-as) [inline src/Docnet.Tests.Integration/PageReaderTests.cs:236]: can we also add a test where multiple flags are used at once?

Comment (bseay) [reply src/Docnet.Tests.Integration/PageReaderTests.cs:246]: I added a test when the RenderAnnotations and Grayscale flags are both set.  I also added a baseline test to verify that annotations pdf renders as expected when no flags are set.

**Comment by bseay** [comment]
I added the test per your request.

please note that if you include my example, I believe that you'll need to bump the version in the `PackageReference` for `Docnet.Core` in the PdfToImage.csproj file appropriately.

**Comment by bseay** [comment]
Alternatively, I can remove the example file so all you'll need to do is merge in the changes as-is.  Just let me know if there's anything else I can do to make it easier on you.

**Pull request merged**